### PR TITLE
Increase coverage of `IBMQClient` tests

### PR DESCRIPTION
**Pull request by Max-Reuter**
### Summary
[WIP] Addresses #93.

### CODE REVIEW

File: test/ibmq/test_ibmq_client_v2.py [hunk 1]
```diff
@@ -298,3 +231,163 @@ def test_url_unreachable(self, qe_token, qe_url):
...
+
+    def test_job_final_status_websocket(self):
+        """Test getting a job's final status via websocket."""
+        result = self.client.job_final_status_websocket(self.job_id)
+        status_present = 'status' in result
+        self.assertTrue(status_present)
+
+    def test_job_properties(self):
+        """Test getting job properties."""
+        # TODO - is {} an acceptable response here?
```
Comment (diego-plan9) [inline test/ibmq/test_ibmq_client_v2.py]: Yes - `properties` might not be available until the job has finished. Actually, can you get the job to finish inside this test, and after that do the relevant assertions (you can use a field that is required for the `BackendProperties` - for example, `backend_name`, probably).

File: test/ibmq/test_ibmq_client_v2.py [hunk 2]
```diff
@@ -298,3 +231,163 @@ def test_url_unreachable(self, qe_token, qe_url):
...
+        result = self.client.job_properties(self.job_id)
+        self.assertIsNotNone(result)
+
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...]
+    def test_job_cancel(self):
```
Comment (diego-plan9) [inline test/ibmq/test_ibmq_client_v2.py]: Can you remove this test? It is also one of those that require slightly trick logic (in this case, you need the job to be in a "cancellable" state) and better of included during #189

File: test/ibmq/test_ibmq_client_v2.py [hunk 3]
```diff
@@ -298,3 +231,163 @@ def test_url_unreachable(self, qe_token, qe_url):
...
+        # TODO
+        pass
+
+    def test_job_get_responsive(self):
+        """Test getting a job responds with something other than None."""
+        result = self.client.job_get(self.job_id)
+        status_present = ['status' in [dict_ for dict_ in result.get('qasms')]]
+        self.assertTrue(status_present)
+
+    def test_job_get_filtered_fields(self):
```
Comment (diego-plan9) [inline test/ibmq/test_ibmq_client_v2.py]: Can you check if this tests is not already covered by the moved `test_get_job_includes_xxx/test_get_job_excludes_xxx` tests a few lines above? Note that `client.get_job()` is actually an alias for `client.job_get()` for compatibility purposes (actually, we should rename the mentioned tests to `test_job_get_includes_....` for consistency).

File: test/ibmq/test_ibmq_client_v2.py [hunk 4]
```diff
@@ -298,3 +231,163 @@ def test_url_unreachable(self, qe_token, qe_url):
...
+    @skip
+    # TODO: RequestsApiError: 404 Client Error: Not Found for url: [...] None, Error code: NotFound.
+    def test_job_result_object_storage(self):
+        """Test getting a job's result using object storage."""
+        # result = self.client.job_result_object_storage(self.job_id)
+
+        # TODO
+        pass
+
+    def test_job_get_responsive(self):
```
Comment (diego-plan9) [inline test/ibmq/test_ibmq_client_v2.py]: Can you rename this one to `test_job_get`, and also relax the assert?

File: test/ibmq/test_ibmq_client_v2.py (hunk 3 above)
Comment (Max-Reuter) [reply test/ibmq/test_ibmq_client_v2.py]: Those tests test `get_job_xxx`, not `job_get_xxx`. I can update them to do the latter, or if we need both tests I'll keep the similar ones currently in the PR.

**Pull request merged**